config/application.rb

```diff
@@ -6,6 +6,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+Dotenv::Railtie.load
```

Comment:
Did you need this for some reason?
Gem's railties are automatically loaded by Rails. That's the whole point of them
And dotenv is usually just a matter of adding to the gemfile